backend/manager/modules/vdsbroker/src/main/java/org/ovirt/engine/core/vdsbroker/monitoring/HostMonitoring.java

```diff
@@ -915,7 +915,7 @@ public static boolean refreshCommitedMemory(VDS host, Map<Guid, VMStatus> vmIdTo
             }
         }
 
-        resourceManager.getVdsManager(host.getId()).setMinRequiredSharedCpusCount(maxSharedCpus);
+        resourceManager.getVdsManager(host.getId()).setMinRequiredSharedCpusCount(maxSharedCores);
```

Comment:
how about renaming setMinRequiredSharedCpusCount to setMinRequiredSharedCoresCount and also CPUPolicyUnit#countMaxRunningVmsSharedCpuCount to countMaxRunningVmsSharedCoreCount (and local variables there)?

Reply:
I do not think we should rename `setMinRequiredSharedCpusCount` to `setMinRequiredSharedCoresCount` - the variable indicates how many CPUs should remain free on the host to accommodate already running VMs and the term CPU can be either thread or core based on countThreadsAsCores. When we're not sure. lets use the term "CPU" as "core" is more specific and not really correct when we count threads.